src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -1004,5 +1005,23 @@ public static class MsalError
         /// <para>Mitigation</para>For troubleshooting details, see https://aka.ms/msal-net-wam .
         /// </summary>
         public const string InitializeProcessSecurityError = "initialize_process_security_error";
+
+        /// <summary>
+        /// <para>What happens?</para>The token cache already contains a token with an OBO cache key that
+        /// matches the <c>longRunningProcessSessionKey</c> passed into <see cref="IConfidentialClientApplication.AcquireTokenInLongRunningProcess"/>.
+        /// <para>Mitigation</para>Call <see cref="IConfidentialClientApplication.InitiateLongRunningProcessInWebApi"/> with a new <c>longRunningProcessSessionKey</c>
+        /// that does not exist in the token cache or call <see cref="IConfidentialClientApplication.AcquireTokenInLongRunningProcess"/> with an already used
+        /// <c>longRunningProcessSessionKey</c>.
+        /// </summary>
+        public const string OboCacheKeyAlreadyInCacheError = "obo_cache_key_in_cache_error";
```

Comment:
I think it would be more clear to add the word "already" to the error code. Looks very similar to the other.
maybe "obo_cache_key_in_cache_already_error"